index.html

```diff
@@ -92,25 +92,25 @@
         <div class="row">
           <div class="col-xs-6 col-md-3">
             <a href="https://medium.com/@gregory.lt.wong/freecodecamp-hong-kong-august-meet-up-recap-intro-to-php-913b4506a26f" class="thumbnail">
-              <img src="https://cdn-images-1.medium.com/max/1600/1*hnrIcmlazqwWqRJAwxvs3g.jpeg" alt="">
+              <img src="https://cdn-images-1.medium.com/max/1600/1*hnrIcmlazqwWqRJAwxvs3g.jpeg" alt="August 2017 meetup - Jeff teaching PhP basics">
               <p class="title">Hong Kong — August Meet Up Recap (Intro to PHP)</p>
             </a>
           </div>
           <div class="col-xs-6 col-md-3">
             <a href="https://medium.com/@gregory.lt.wong/freecodecamp-hong-kong-july-meet-up-recap-e3446f09858e" class="thumbnail">
-              <img src="https://cdn-images-1.medium.com/max/1600/1*wZZ0Fh6Gm7nS5GTbcnylOg.jpeg" alt="">
+              <img src="https://cdn-images-1.medium.com/max/1600/1*wZZ0Fh6Gm7nS5GTbcnylOg.jpeg" alt="July 2017 meetup - casual sharing">
               <p class="title">freeCodeCamp Hong Kong - July Meet Up Recap</p>
             </a>
           </div>
           <div class="col-xs-6 col-md-3">
             <a href="https://medium.com/@gregory.lt.wong/freecodecamp-hong-kong-large-scale-community-building-855fb642b0e6" class="thumbnail">
-              <img src="https://cdn-images-1.medium.com/max/2000/1*dDIcbxAeUfIy3FQUtFcbKQ.jpeg" alt="">
+              <img src="https://cdn-images-1.medium.com/max/2000/1*dDIcbxAeUfIy3FQUtFcbKQ.jpeg" alt="June 2017 meetup - casual sharing">
               <p class="title">freeCodeCamp Hong Kong - Large Scale Community Building</p>
             </a>
           </div>
           <div class="col-xs-6 col-md-3">
             <a href="https://medium.com/@gregory.lt.wong/freecodecamp-hong-kong-june-meet-up-recap-ebd833a76291" class="thumbnail">
-              <img src="https://cdn-images-1.medium.com/max/2000/1*gd1mg6pCVrrHhbmMUlNZZg.jpeg" alt="">
+              <img src="https://cdn-images-1.medium.com/max/2000/1*gd1mg6pCVrrHhbmMUlNZZg.jpeg" alt="June 2017 meetup - First meetup @Accelerate">
```

Comment:
Would you mind to add a space between `@` and `Accelerate` so that it wouldn't be mistaken as a username or something..? :)